renderer/views/player.js

```diff
@@ -23,10 +23,16 @@ function Player (state, dispatch) {
     state.video.duration = videoElement.duration
   }
 
+  // When in fullscreen, hide player controls if the mouse stays still for a while
+  //console.log('STATIONARY FOR ' + new Date().getTime() - state.video.mouseStationarySince)
```

Comment:
We should hide the player controls if the mouse is still, even when not in fullscreen.

YouTube, Quicktime do the same.